src/tests/functional/plugin/conformance/test_runner/conformance_infra/include/gflag_config.hpp

```diff
@@ -72,6 +75,7 @@ static void showUsage() {
     std::cout << "    --output_folder \"<path>\"         " << output_folder_message << std::endl;
     std::cout << "    --plugin_lib_name                " << output_folder_message << std::endl;
     std::cout << "    --shape_mode  \"<value>\"          " << shape_mode_message << std::endl;
+    std::cout << "    --test_timeout                " << test_timeout_message << std::endl;
```

Comment:
Please check, that the message is aligned in terminal (use `--help` option)

Reply:
aligned